diff --git a/site-lisp/vterm-repl-aider.el b/site-lisp/vterm-repl-aider.el
index c902427..97df4a3 100644
--- a/site-lisp/vterm-repl-aider.el
+++ b/site-lisp/vterm-repl-aider.el
@@ -153,7 +153,7 @@
       "--multiline")
     "the available command arguments used by aider")
 
-(defvar vtr-aider-cmd "aider" "the command used to start the aider")
+(defvar vtr-aider-cmd "fhs aider" "the command used to start the aider")
 (defvar vtr-aider-args "--watch-files" "the arguments used to start the aider")
 
 (defun vtr-aider-full-command ()
diff --git a/site-lisp/vterm-repl.el b/site-lisp/vterm-repl.el
index a346204..94f36c0 100644
--- a/site-lisp/vterm-repl.el
+++ b/site-lisp/vterm-repl.el
@@ -153,7 +153,7 @@ the window with that number as a suffix." repl-name)
 (vtr-create-schema "aichat" "aichat -s" :bracketed-paste-p t)
 
 ;;;###autoload (autoload #'vtr~ipython-start "vterm-repl" nil t)
-(vtr-create-schema "ipython" "ipython" :bracketed-paste-p t)
+(vtr-create-schema "ipython" "fhs ipython" :bracketed-paste-p t)
 
 ;;;###autoload (autoload #'vtr~radian-start "vterm-repl" nil t)
 (vtr-create-schema "radian" "radian" :bracketed-paste-p t :end-pattern "")
